packages/block-editor/src/utils/transform-styles/index.js

```diff
@@ -18,45 +64,14 @@ import rebaseUrl from 'postcss-urlrebase';
  * @return {Array} converted rules.
  */
 const transformStyles = ( styles, wrapperSelector = '' ) => {
```

Comment:
As mentioned in person, if we're using an iframe, transforming styles might not be necessary anymore assuming we also pass the "url" to the stylesheet in addition to its content. We could just use the URL to load the stylesheets within the iframe.

Reply:
Right, I'll try this next. This PR is still useful for an editor without iframe though (when there's a registered block that is not v3)